src/components/edit/earthEngine/BandSelect.jsx

```diff
@@ -7,21 +6,24 @@ import { SelectField } from '../../core/index.js'
 
 const DEFAULT_NO_BAND = []
 
-const BandSelect = ({ band = DEFAULT_NO_BAND, bands, setBand, errorText }) => (
-    <SelectField
-        label={i18n.t('Groups')}
-        items={bands}
-        multiple={true}
-        value={band}
-        onChange={setBand}
-        errorText={errorText}
-    />
-)
+const BandSelect = ({ band, bands, setBand, errorText }) => {
+    const value = band ?? bands?.default ?? DEFAULT_NO_BAND
```

Comment:
Line 10 tests whether bands exists, but in the return SelectField, bands is assumed to exist.  In propTypes, bands is a required object